htdocs/comm/propal/card.php

```diff
@@ -124,7 +124,7 @@
 $usercancreateinvoice = $user->rights->facture->creer;
 $usercancreatecontract = $user->rights->contrat->creer;
 $usercancreateintervention = $user->hasRight('ficheinter', 'creer');
-$usercancreatepurchaseorder = ($user->rights->fournisseur->commande->creer || $user->rights->supplier_order->creer);
+$usercancreatepurchaseorder = ($user->hasRight('fournisseur', 'creer') || $user->hasRight('supplier_order', 'creer'));
```

Comment:
The permission $user->rights->fournisseur->commande->creer is on 3 levels instead of 2, so you must use
$user->hasRight('fournisseur', 'commande', 'creer')

Reply:
@eldy ok it's done